src/results/generator.jl

```diff
@@ -86,15 +86,15 @@ function add_generator_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _
 	r["year_one_to_load_series_kw"] = round.(value.(generatorToLoad), digits=3)
 
     GeneratorFuelUsed = @expression(m, sum(m[:dvFuelUsage][t, ts] for t in p.techs.gen, ts in p.time_steps))
-	r["fuel_used_gal"] = round(value(GeneratorFuelUsed), digits=2)
+	r["average_annual_fuel_used_gal"] = round(value(GeneratorFuelUsed), digits=2)
```

Comment:
`average_annual_fuel_used_gal` is not really an average right? if not I suggest removing `average_`.

Reply:
"average" to me implies that we had more than one value that we averaged over the analysis years, but we only have one value and it is the same every year. so technically it is the average - I just find it misleading. we can leave it.